src/algo/depth-to-rgb-calibration/optimizer.cpp

```diff
@@ -122,6 +122,14 @@ namespace
 optimizer::optimizer( settings const & s )
     : _settings( s )
 {
+    if (_settings.is_manual_trigger)
+    {
+        adjust_params_to_manual_mode();
+    }
+    else
+    {
+        adjust_params_to_apd_gain(_settings.ambient);
```

Comment:
It's kind of weird that the opposite to manual mode is apd gain.
Also, the dependence on APD should be inside the function, and not passed in, no? We have access to any setting we need -- why pass it in by value?